### PR TITLE
Expand checklist and clarify requirements

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Expand checklist and clarify requirements

### CODE REVIEW

**Review by tadeboro — APPROVED** [review]

**Comment by felixfontein** [comment]
Since this does change rules, I think we need to vote on this. (I'm +1 on this change btw, if someone wonders :) .)

**Comment by Andersson007** [comment]
created a topic https://github.com/ansible-community/community-topics/issues/67

**Review by ssbarnea — APPROVED** [review]

**Review by jillr — APPROVED** [review]

File: collection_checklist.md [hunk 1]
```diff
@@ -33,8 +33,11 @@ Every comment should say whether the reviewer expects it to be addressed, or whe
 - [ ] collection dependencies must have a lower bound on the version which is at least 1.0.0, and are all part of the `ansible` package
 - [ ] `meta/runtime.yml` defines the minimal version of Ansible which the collection works with
 - [ ] has changelog, preferably with `changelogs/changelog.yaml`
+- [ ] collection repository should not contain any large objects (binaries) comparatively to the current Galaxy tarball size limit of 20 MB like, for example, package installers for testing purposes
```
Comment (gundalow) [inline collection_checklist.md:36]:
```suggestion
- [ ] collection repository should not contain any large objects (binaries) comparatively to the current Galaxy tarball size limit of 20 MB. For example, package installers for testing purposes shouldn't be added."
```

File: collection_checklist.md [hunk 2]
```diff
@@ -33,8 +33,11 @@ Every comment should say whether the reviewer expects it to be addressed, or whe
...
+- [ ] collection repository should not contain any unnecessary files like, for example, temporary files created by development tools
```
Comment (gundalow) [inline collection_checklist.md:37]:
```suggestion
- [ ] collection repository should not contain any unnecessary files like, for example, temporary files created by development tools. Temporary files MUST be added to `.gitignore`.
```

**Review by gundalow — COMMENTED** [review]
Looks good, thank you.

File: collection_checklist.md [hunk 3]
```diff
@@ -33,8 +33,11 @@ Every comment should say whether the reviewer expects it to be addressed, or whe
 - [ ] collection dependencies must have a lower bound on the version which is at least 1.0.0, and are all part of the `ansible` package
 - [ ] `meta/runtime.yml` defines the minimal version of Ansible which the collection works with
 - [ ] has changelog, preferably with `changelogs/changelog.yaml`
+- [ ] collection repository should not contain any large objects (binaries) comparatively to the current Galaxy tarball size limit of 20 MB. For example, package installers for testing purposes shouldn't be added."
```
Comment (Andersson007) [inline collection_checklist.md:36]:
```suggestion
- [ ] collection repository should not contain any large objects (binaries) comparatively to the current Galaxy tarball size limit of 20 MB. For example, package installers for testing purposes shouldn't be added.
```

**Review by gundalow — APPROVED** [review]

**Pull request merged**